chromium_src/chrome/common/channel_info_posix.cc

```diff
@@ -22,6 +22,30 @@ std::string GetChannelSuffixForDataDir() {
   return std::string();
 }
 
+#if BUILDFLAG(IS_LINUX)
+std::string GetChannelSuffixForExtraFlagsEnvVarName() {
+#if BUILDFLAG(OFFICIAL_BUILD)
+  const auto channel_state = GetChannelImpl();
+  switch (channel_state.channel) {
+    case version_info::Channel::DEV:
```

Comment:
do we actually want this? I don't know what they use it for

Reply:
@cdesouza-chromium ☝️

Reply:
To be honest, I also have no idea what this is for. I merely copy and pasted the upstream implementation that was getting shadowed and causing linking errors on posix.

Reply:
The upstream commit says: 
<pre>
This aims to provide a standard system-wide mechanism to set these
additional arguments to chrome via a set of environment variables which,
in *nix world, is a well-known and widely used mechanism for use-cases
like this.
</pre>

Reply:
Yeah, I had seen that, but what does it mean? Is this relevant to us? Or should we change the environment variable to use `BRAVE_` prefix down below?

Reply:
Not sure, we seem to have kept `CHROME_DESKTOP` in the `GetDesktopName` function below and we have not replaced `kExtraFlagsVarName` in [chrome/app/chrome_main_linux.cc](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:chrome/app/chrome_main_linux.cc;l=37)

Reply:
@fmarier, as a linux user, is this something that would be useful to have?

Reply:
I also have no idea how this is used in practice.

Reply:
well, I guess it probably doesn't hurt anything to leave it for now